setup.py

```diff
@@ -137,9 +137,15 @@ def get_release_info():
     'Topic :: Software Development :: Widget Sets',
 ]
 
+# Unfortunately, distutils.version.StrictVersion does not support semantic
+# versioning (www.semver.org) for pre-releases. This breaks, e.g,  bdist_msi
+# As a workaround, we normalize the version to be acceptable for StrictVersion
+# Note that the normalized version is identical to the semver except for
+# pre-releases (where '-dev' is replaced by 'a0').
+_version = release.version.replace('-dev', 'a0')
```

Comment:
> Isn't it possible to change it directly on the bumpversion configuration level?

It is definitely possible (and easy). But I will try to justify my decision: before implementing my proposed solution I considered the following options:

1.- (what @reszelaz suggests) use the distutils version style (e.g. "4.0.3a0") instead of the semver style ("4.0.3-dev") in the `taurus.core.release.version` and configure bumpversion to update it directly. This has the advantage of not differentiating between the taurus **module** version and the taurus **distribution** version. But I rejected it because that would mean giving up on semver, which I think it is not worth.

2.- limit the scope of the normalization of the version to the bdist_msi command. This can be done by monkey patching distutils from setup.py (see [this](http://bugs.python.org/issue6040#msg133094)). I rejected this approach because it implies polluting our setup.py with potentially fragile code and it results in an inconsistent version naming between different  distributions of the same source (e.g., the .egg, .wheel or .tar.gz would use semver while the .msi would use distutils style)
 
The main drawback of my proposed solution is that it makes a difference between the **module** version and the **distribution** version (i.e., after installing **a prerelease** taurus, you would get a different answer if you interrogate `taurus.Release.version` than if you interrogate `pkg_resources.require("taurus")[0].version`). But note that this is the case already, and that this only happens on pre-release installations, and can be understood if you consider that module != distribution